### PR TITLE
Python 3 for files without a .py extension

### PR DESCRIPTION
**Pull request by cclauss**
$ `2to3 --fix=print wskadmin`

The flake8 linter does not pick up these files because their filenames do not end in .py

### CODE REVIEW

**Comment by markusthoemmes** [comment]
Can we integrate this linter into our repository? Changing these files without bringing the necessary automation will just break after a while and get rotten.

**Comment by cclauss** [comment]
I lint the OpenWhisk files by pulling them into a [separate repo with a Python oriented .travis.yml](https://github.com/cclauss/flake8_for_ow/blob/master/.travis.yml) file.  If someone can figure out how to do a Python test matrix inside a Scala oriented .travis.yml file, that would be cool.  Flake 8 changes behavior depending on which version of Python that it is running on so the Python 2 / Python 3 matrix is useful.  However if we could only get Python 3, that would be sufficient for our purposes.

**Comment by markusthoemmes** [comment]
Can't you just install flake8 and run it with appropriate settings?

**Comment by cclauss** [comment]
Flake8 is [already in .travis.yml](https://travis-ci.org/openwhisk/openwhisk/builds) but that runs under Python 2.7 so it will not find Python 3 syntax errors.  I could not figure out is how to install Python 3 before running flake8.

**Comment by cclauss** [comment]
Upgrading .travis.yml to use **dist: trusty** will give us Python 3.5.2 which is good enough to run flake8.  https://github.com/openwhisk/openwhisk/pull/2020/files#diff-354f30a63fb0907d4ad57269548329e3R3

**Comment by cclauss** [comment]
OK.  We have flake8 running under Python 3.  The first run is under the travis `before_script` twisty runs with the --zero-exit option so it treats _everything_ as warnings.  It currently finds 59 style related issues.  The second run is in the travis `script` section and halts the build on Python syntax errors.

Ready for review.

File: tools/admin/wskadmin [hunk 1]
```diff
@@ -116,9 +119,10 @@ def parseArgs():
     propmenu = subparsers.add_parser('syslog', help='work with system logs')
     subparser = propmenu.add_subparsers(title='available commands', dest='subcmd')
 
-    subcmd = subparser.add_parser('get', help='get logs for a transaction id')
-    subcmd.add_argument('tid', help='the transaction id')
-    subcmd.add_argument('component', help='component, one of [controller, invokerN] where N is invoker index')
+    subcmd = subparser.add_parser('get', help='get logs')
```
Comment (rabbah) [inline tools/admin/wskadmin]: if you do 

```
git fetch --all 
git rebase upstream/master
```
then you pick up the latest changes without making them appear as new changes in the pr.
@houshengbo wrote up a nice tutorial that might be helpful in general.

**Comment by rabbah** [comment]
This branch has a conflict despite what github says (on the first commit when attempting to rebase). @cclauss do you mind rebasing this to upstream/master?

**Comment by rabbah** [comment]
Squashed and merged as https://github.com/openwhisk/openwhisk/commit/4ea7833815356ff18b2d4df93bf20f3f000c7ccb.

PG1/1289 approved.

**Pull request closed without merging**